CefSharp.props

```diff
@@ -5,14 +5,14 @@
     <VisualStudioProductVersion>2013</VisualStudioProductVersion>
     <VisualStudioProductVersion Condition="'$(VisualStudioVersion)'=='14.0'">2015</VisualStudioProductVersion>
     <VisualStudioProductVersion Condition="'$(VisualStudioVersion)'=='15.0'">2017</VisualStudioProductVersion>
-    <VisualStudioProductVersion Condition="'$(VisualStudioVersion)'=='16.0'">2017</VisualStudioProductVersion>
+    <VisualStudioProductVersion Condition="'$(VisualStudioVersion)'=='16.0'">2019</VisualStudioProductVersion>
```

Comment:
This is where things get complicated, the default `cef.sdk` package only has `VS2015` and `VS2017` versions of the `wrapper` by default. With this change anyone using the packages from `Nuget.org` would be unable to build using `VS2019`

The `VisualStudioProductVersion` variable is used to determine which directory to load the `.lib` file from.

https://github.com/cefsharp/CefSharp/blob/a843407dab56741263f6783c368f85a7bb720c15/CefSharp.Core/CefSharp.Core.vcxproj#L136

Maybe it's possible to have `AppVeyor` build `VS2015`, `VS2017` and `VS2019` versions of the wrapper. See https://www.appveyor.com/docs/windows-images-software/ for list of software installed by image. 

Current image is https://github.com/cefsharp/cef-binary/blob/master/appveyor.yml#L1

Reply:
Ah, I could add the v142 to the default for cef-binary building but yeah we would have to change the appveyor default to vs2019 version.  Now it makes sense why you had the newer vs versions targeting the older option.

Reply:
`VS2019` appears perfectly capable of compiling with a `v141` lib file (as was promised by `Microsoft` that they'd be compatible).

I understand from your `Docker` build scripts you'd be looking to build everything with `VS2019`.

Open to suggestions here.

Reply:
Targeting v141 is no problem.  With docker could even install V2017 and V2019 side by side without issue.   In addition can easily patch in any changes to the code instead to manually change to v142.   Keeping in on V141 for now would seem the best solution.